Yoast/YoastSEO.js
# Refactor meta description keyword assessment

## Summary

This PR can be summarized in the following changelog entry:

* Changes the meta description keyword assessment. The assessment now takes morphological forms of each term in the keyword/phrase into account. The scoring scheme has been changed to check if keyword terms are used in the same sentence.

**Note**: Score texts will need to be changed.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Open or create a post in the WordPress editor.
* Enter a couple of focus keywords in its respective box in the SEO analysis section of the plugin.
    * including stop words and other non-content words.
* Enter a meta description...
    * with all of the content keywords in the same sentence. This should give you a GOOD score on the `metaDescriptionKeywordAssessment`.
    * with three sentences containing all of the content keywords each. This should give you a BAD score.
    * with all of the content words, but not in the same sentence. This should give you an OK score.
    * with no content words at all. This should give you a BAD score.

Fixes #1754
